firefox-ios/Client/Frontend/Browser/Toolbars/Redux/ToolbarMiddleware.swift

```diff
@@ -145,7 +145,7 @@ class ToolbarMiddleware: FeatureFlaggable {
         case .tap:
             handleToolbarButtonTapActions(action: action, state: state)
         case .longPress:
-            handleToolbarButtonLongPressActions(actionType: buttonType, windowUUID: uuid)
+            handleToolbarButtonLongPressActions(action: action, windowUUID: uuid)
```

Comment:
Nit: If we are passing the action already then we don't need the `windowUUID` anymore as the action has it as well.